CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2022-05-17
```

Comment:
When adding new changelog items, you can just add them on top without this date header. We only add that when we're doing the actual release.